webaudio-audionode-disconnect/index.html

```diff
@@ -0,0 +1,134 @@
+
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>WebAudio AudioNode selective disconnect Sample</title>
+
+    <!-- Add to homescreen for Chrome on Android -->
+    <meta name="mobile-web-app-capable" content="yes">
+    <link rel="icon" sizes="192x192" href="../images/touch/chrome-touch-icon-192x192.png">
+
+    <!-- Add to homescreen for Safari on iOS -->
+    <meta name="apple-mobile-web-app-title" content="WebAudio AudioNode selective disconnect Sample">
+
+    <meta name="apple-mobile-web-app-capable" content="yes">
+    <meta name="apple-mobile-web-app-status-bar-style" content="black">
+    <link rel="apple-touch-icon-precomposed" href="../images/apple-touch-icon-precomposed.png">
+
+    <!-- Tile icon for Win8 (144x144 + tile color) -->
+    <meta name="msapplication-TileImage" content="images/touch/ms-touch-icon-144x144-precomposed.png">
+    <meta name="msapplication-TileColor" content="#3372DF">
+
+    <link rel="icon" href="../images/favicon.ico">
+
+    <link rel="stylesheet" href="../styles/main.css">
+  </head>
+
+  <body>
+    <h1>WebAudio AudioNode selective disconnect Sample</h1>
+
+    <p>Available in <a href="http://www.chromestatus.com/feature/4793068276416512">Chrome 43+</a></p>
+
+    <p>
+      The overloaded method <code>AudioNode.disconnect</code> supports disconnecting either all connections, or a single connection (see the <a href="https://webaudio.github.io/web-audio-api/#idl-def-AudioNode">audio node spec</a>).
+      Selective disconnection, choosing a specific signal path to disconnect, is supported as of Chrome 43+.
+    </p>
+
+    <div class="output">
+      <div>
+        Plays a demo audio file, but only connects the right channel.
+        The left channel is toggled via <code>AudioNode.connect</code> and <code>AudioNode.disconnect</code> with selective arguments.
+      </div>
+      <button id="btn-play">Play</button>
+      <button id="btn-connect">Toggle Left Channel</button>
+    </div>
+
+    <!-- // [START code-block] -->
+    <script>
+
+    var context = new AudioContext();
+
+    // Load the audio file at the specified url, returning a Promise containing
+    // an AudioBuffer.
+    function loadAudio(url) {
+      // fetch is available in Chrome 42+.
+      return fetch(url).then(function(response) {
+        return response.arrayBuffer();
+      }).then(function(data) {
+        return new Promise(function(resolve, reject) {
+          context.decodeAudioData(data, function(buffer) {
+            buffer ? resolve(buffer) : reject();
+          });
+        });
+      })
+    }
+
+    var playerNode = context.createBufferSource();
+    playerNode.loop = true;
+
+    document.getElementById('btn-play').addEventListener('click', function() {
+      this.disabled = true;
+      playerNode.start();
+    });
+
+    loadAudio("../audio/techno.wav").then(function(buffer) {
+      playerNode.source = buffer;
+    }, function(e) {
```

Comment:
See my comment in the other PR about using `catch()` instead of the two-argument `then()`, since it's arguably clearer to developers who don't know about promises.